### PR TITLE
Extended interface to enable opening by description, serial or index,…

### PR DESCRIPTION
**Pull request by marogi**
… fixed segfault with libftdi 1.4

### CODE REVIEW

File: ft232/ft232defines.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python
```
Comment (mandross) [inline ft232/ft232defines.py:1]: do we need shebang here?

**Review by mandross — COMMENTED** [review]
Shall we split it into two pull request to  get the clean history? The author of the lib may not want to merge everything.

File: ft232/ft232defines.py (hunk 1 above)
Comment (marogi) [reply ft232/ft232defines.py:1]: Will remove

**Comment by marogi** [comment]
Closing this PR to split it into two separate

**Pull request closed without merging**